### PR TITLE
Add Color and Nodesize legend

### LINKED ISSUE #63: create a color legend
so far the node color can be assigned to different attributes. However, there is no visual indication what a specific color means. A color legend will help with that

### PR DESCRIPTION
**Pull request by sgratzl**
closes #63 

added a color and node size legend to the left bottom.

image

image

tho, especially in higher zoom levels, the node size legend becomes really big and less informative.

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -214,6 +215,88 @@ export class ApplicationStore {
             }
         });
     }
+
+    @computed
+    get degree2SizeFunc() {
```
Comment (waxlamp) [inline src/store/index.js]: s/2/To/

File: src/store/index.js [hunk 2]
```diff
@@ -214,6 +215,88 @@ export class ApplicationStore {
...
+            default:
+                return (v) => typeof v === 'number' ? v.toFixed(3) : v;
+        }
+    }
+
+
+    static COLOR_SCALE = scaleSequential(interpolateViridis);
+    static INVALID_VALUE_COLOR = '#ff0000';
+    static EXISTS_COLOR = 'rgb(81,96,204)';
+    static NOT_EXISTENT_COLOR = '#de2d26';
```
Comment (waxlamp) [inline src/store/index.js]: Let's rename these to `DISCOVERED_COLOR` and `UNDISCOVERED_COLOR`.

File: src/geojs-scene-manager/index.js [hunk 3]
```diff
@@ -210,44 +209,13 @@ export class GeoJSSceneManager {
     this.map.draw();
   }
 
-  hideAfter () {}
-
-  setDegreeSize (year, level) {
+  hideAfter() { }
```
Comment (waxlamp) [inline src/geojs-scene-manager/index.js]: Remove the empty space between the braces.

File: src/geojs-scene-manager/index.js [hunk 4]
```diff
@@ -210,44 +209,13 @@ export class GeoJSSceneManager {
...
+
```
Comment (waxlamp) [inline src/geojs-scene-manager/index.js]: Remove whitespace.

File: src/store/index.js (hunk 1 above)
Comment (sgratzl) [reply src/store/index.js]: I guess it should mean `degree2SizeFunc` -> `degreeToSizeFunc`

**Comment by sgratzl** [comment]
feedback is integrated

**Pull request merged**